kclvm/tools/src/LSP/src/completion.rs

```diff
@@ -67,30 +74,70 @@ fn func_ty_complete_label(func_name: &String, func_type: &FunctionType) -> Strin
     )
 }
 
+fn ty_complete_label(ty: &Type) -> Vec<String> {
+    match &ty.kind {
+        kclvm_sema::ty::TypeKind::Bool => vec!["True".to_string(), "False".to_string()],
+        kclvm_sema::ty::TypeKind::BoolLit(b) => {
+            vec![if *b {
+                "True".to_string()
+            } else {
+                "False".to_string()
+            }]
+        }
+        kclvm_sema::ty::TypeKind::IntLit(i) => vec![i.to_string()],
+        kclvm_sema::ty::TypeKind::FloatLit(f) => vec![f.to_string()],
+        kclvm_sema::ty::TypeKind::Str => vec![r#""""#.to_string()],
+        kclvm_sema::ty::TypeKind::StrLit(s) => vec![format!("{:?}", s)],
+        kclvm_sema::ty::TypeKind::List(_) => vec!["[]".to_string()],
+        kclvm_sema::ty::TypeKind::Dict(_) => vec!["{}".to_string()],
+        kclvm_sema::ty::TypeKind::Union(types) => {
+            types.iter().flat_map(|ty| ty_complete_label(ty)).collect()
+        }
+        kclvm_sema::ty::TypeKind::Schema(schema) => {
+            vec![format!(
+                "{}{}{}",
+                if schema.pkgpath.is_empty() || schema.pkgpath == MAIN_PKG {
+                    "".to_string()
+                } else {
+                    format!("{}.", schema.pkgpath.split(".").last().unwrap())
+                },
+                schema.name,
+                "{}"
+            )]
+        }
+        _ => vec![],
+    }
+}
+
 /// Computes completions at the given position.
 pub(crate) fn completion(
     trigger_character: Option<char>,
     program: &Program,
     pos: &KCLPos,
     prog_scope: &ProgramScope,
 ) -> Option<lsp_types::CompletionResponse> {
-    if let Some('.') = trigger_character {
-        completion_dot(program, pos, prog_scope)
-    } else {
-        let mut completions: IndexSet<KCLCompletionItem> = IndexSet::new();
+    match trigger_character {
+        Some(c) => match c {
+            '.' => completion_dot(program, pos, prog_scope),
+            '=' | ':' => completion_assign(program, pos, prog_scope),
```

Comment:
Why do we need to use `:` and `=` to trigger completion? Personally, I believe that after entering `:` and `=`, a space ` ` will be entered, and completion will be invalidated. I think it should be something that ordinary character completion should handle.

Reply:
use `:` or `=` to trigger completion for schema attr config, when entering `:` or `=`, this will complete a space and the following content. If entering space after trigger, user also can use ordinary character completion like variable, pkg, schema name